xml/System.Text.Json/JsonWriterOptions.xml

```diff
@@ -55,9 +60,17 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Defines whether the <see cref="T:System.Text.Json.Utf8JsonWriter" /> should skip structural validation and allow
+            the user to write invalid JSON, when set to true. If set to false, any attempts to write invalid JSON will result in
+            a <exception cref="T:System.InvalidOperationException" /> to be thrown.</summary>
```

Comment:
I don't know which way this renders... "a System.InvalidOperationException" or "an InvalidOperationException".  Silly a vs an.

Reply:
I'll double check. This was actually using the wrong tag so the link wasn't even appearing.